### DISCUSSION TITLE
请问如何自行更改某个功能的代码

### DISCUSSION THREAD

**Discussion by codeNiuMa** [the1812/Bilibili-Evolved#3700] [提问 / Q&A]
有一个视频内显示时间的功能，Commit: e1d9ceac8，内部名称: videoCurTime，但是显示字体较小，通过F12看到可以修改字体大小，可如何在源代码处实现呢？

<span id="cur-time" style="position: absolute; top: 0px; right: 0px; z-index: 10; font-size: x-large（就在这里）; font-weight: bold; margin: 1rem; color: rgb(255, 72, 0); opacity: 0.55;">9:16</span>

**Comment by the1812** [comment]
源代码在另一个仓库: https://github.com/FoundTheWOUT/Bilibili-Evolved/blob/preview/registry/lib/components/video/player/localtime/index.ts

开发方式可以参考 https://github.com/the1812/Bilibili-Evolved/blob/master/CONTRIBUTING.md
